src/ServiceControl.Monitoring/Infrastructure/EndpointInputQueue.cs

```diff
@@ -13,47 +13,10 @@ public EndpointInputQueue(string endpointName, string inputQueue)
         public string EndpointName { get; }
         public string InputQueue { get; }
 
-        public bool Equals(EndpointInputQueue other)
-        {
-            if (other is null)
-            {
-                return false;
-            }
-
-            if (ReferenceEquals(this, other))
-            {
-                return true;
-            }
-
-            return string.Equals(EndpointName, other.EndpointName) && string.Equals(InputQueue, other.InputQueue);
-        }
-
-        public override bool Equals(object obj)
-        {
-            if (obj is null)
-            {
-                return false;
-            }
+        public bool Equals(EndpointInputQueue other) => string.Equals(EndpointName, other.EndpointName) && string.Equals(InputQueue, other.InputQueue);
```

Comment:
This is going to require an exact case match. So did the original so this isn't a new problem.